vertical-pod-autoscaler/go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	golang.org/x/time v0.4.0
 	k8s.io/api v0.28.3
 	k8s.io/apimachinery v0.28.3
+	k8s.io/apiserver v0.0.0-00010101000000-000000000000
```

Comment:
It's weird that it is using a pseudo-version number... Do you know why it is importing this?

What happens if you try to pull in a specific version? Maybe we can use the same as cluster autoscaler? https://github.com/kubernetes/autoscaler/blob/386f0f759fa6f794b628c5e3bb01601345a82398/cluster-autoscaler/go.mod#L47

(I'm not an expert in this so sorry if this is totally off - this just looks weird to me)

Reply:
I think 679d6bde74 fixes it
I used 0.28.3, since that's what the VPA currently uses